packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -505,39 +1056,29 @@ <h2>Ranges</h2>
         return tooltips[0];
       }
 
+      // TODO: Confirm if calculating this every mousemove has a negligible cost. Or do it just once and save it as a data property in the slider container (instead of its parent)
+      function getIsVerticalSlider(sliderContainer) {
+        return sliderContainer.parentNode.classList.contains("iui-slider-vertical");
+      }
```

Comment:
@mayank99 Can you please let me know if calling this method on every thumb movement is okay? Or should I save a data attribute of the orientation in the thumb also? (Currently, the orientation class of `iui-slider-vertical` is only in `iui-slider-component-container`) I am leaning towards the first option because with the second, I feel there is a source of inconsistency between the orientation data attribute in the `iui-slider-component-container` and the `iui-slider-thumb`.

Reply:
It is demo/test html so don't worry, also it shouldn't be that costly to call it constantly.

Reply:
**[DONE]** Thanks, then will leave it as it is